miden-vm/tests/integration/exec_iters.rs

```diff
@@ -76,7 +76,7 @@ fn test_exec_iter() {
             asmop: Some(AsmOpInfo::new(
                 AssemblyOp::new(
                     mem_storew1_loc.clone(),
-                    "#exec::#main".to_string(),
+                    "$exec::#main".to_string(),
```

Comment:
Question: should we change `#main` to `$main` as well?

Reply:
Good catch, yes, same issue with that one as well